bokeh/charts/chart.py

```diff
@@ -82,14 +82,14 @@ class Chart(Plot):
     def __init__(self):
 
         # Initializes then gets default properties
-        super(Chart, self).__init__(id=self._options.id or make_id())
+        # NOTE: we use option._id here since ChartOptions converts id arg to _id
+        super(Chart, self).__init__(id=self._options._id or make_id())
```

Comment:
I guess the idea is that Model lets you set `id=something` (though I'm not sure why that's ever useful :-)

Another way to do this would be in the `_chained_options` decorator above, pop all kwargs that are in `ChartOptions.properties()`, pass them to `ChartOptions`, then pass the remaining kwargs to the `orig_init`. An advantage of that is that then ChartOptions can be a HasProps instead of a Model, since it isn't meant to go in a Document. Another advantage is that anytime Model grows new kwargs, Chart would automatically support them.